config/config.go

```diff
@@ -86,8 +100,18 @@ func (config *Config) Validate(path string) ([]string, error) {
 func GetOptionalParameters(config *Config, defaultPort string,
 	defaultDataRateMsec, defaultMapRateSec int, logger golog.Logger,
 ) (string, int, int, bool, bool, bool, error) {
+	modularizationV2Enabled := false
+	if config.ModularizationV2Enabled == nil {
+		logger.Debug("no modularization_v2_enabled given, continuing with modularization v1")
+	} else {
+		modularizationV2Enabled = *config.ModularizationV2Enabled
+	}
+
+	// do not validate port if mod v2 is enabled
```

Comment:
[nit] here and below, comments should be formatted as sentences (capitalization and ending with a period)